src/components/codemirror.tsx

```diff
@@ -181,6 +181,21 @@ export default function CodeMirror(props: CodeMirrorProps) {
 				if(props.roomState)
 					props.roomState.value = { ...props.roomState?.value, connectionStatus: ConnectionStatus.CONNECTED };
 		});
+			provider.awareness.on("update", () => {
+				let participants: RoomParticipant[] = [];
+				provider.awareness.getStates().forEach((state) => {
+					try{
+						participants.push({
+							userEmail: state.user.name,
+							isHost: state.user.host
+						})
+					} catch(e){
+						return
```

Comment:
is it OK for this to be very quietly failing w/o any logging?

Reply:
yup. bc that means that the participant doesn't have an email, therefore it's the saving instance and we don't count that towards the participants